sonar-helm-for-iac/build.gradle.kts

```diff
@@ -0,0 +1,90 @@
+plugins {
+    id("org.sonarsource.iac.java-conventions")
+}
+
+description = "SonarSource IaC Analyzer :: Sonar Helm for IaC"
+
+val sonar_sources by extra { "." }
+val sonar_inclusions by extra { "**/*.go" }
+val sonar_exclusions by extra { "**/build/**,**/org.sonarsource.iac.helm/**" }
+val sonar_tests by extra { "." }
+val sonar_test_inclusions by extra { "**/*_test.go" }
+val sonar_go_tests_reportPaths by extra { "build/test-report.out" }
+val sonar_go_coverage_reportPaths by extra { "build/test-coverage.out" }
+
+tasks.register<Exec>("compileProtobufGo") {
+    description = "Compile the Go protobuf."
+    group = "build"
+
+    inputs.files("template-evaluation.proto")
+    outputs.files("org.sonarsource.iac.helm/template-evaluation.pb.go")
+    outputs.cacheIf { true }
+
+    commandLine("protoc", "-I=${project.projectDir}", "-I=" + System.getProperty("user.home") + "/go/protobuf/include",
+    "--go_out=${project.projectDir}", "${project.projectDir}/template-evaluation.proto")
+    doFirst {
+        println("Running command: ${commandLine}")
+    }
+    doLast {
+        println("Compile protobuf done")
+    }
+}
+
+// Define and trigger tasks in this order: clean, compile and test go code
+tasks.register<Exec>("cleanGoCode") {
+    description = "Clean all compiled version of the go code."
+    group = "build"
+
+    callMake(this, "clean")
+    doLast {
+        println("cleanGoCode")
```

Comment:
Do you want to keep this log or was it for debug only? If you want to keep it, I suggest to either make it `doFirst` or log `cleanGoCode done`

Reply:
It was part of some early debug log I put in place to ensure the flow/order of execution, not needed anymore I forgot to clean them, it's removed now.